src/Tes.Runner/Transfer/BlobUploader.cs

```diff
@@ -119,7 +138,7 @@ public async Task<long> UploadAsync(List<UploadInfo> uploadList)
 
         private static void ValidateUploadList(List<UploadInfo> uploadList)
         {
-            ArgumentNullException.ThrowIfNull(uploadList);
+            ArgumentNullException.ThrowIfNull(uploadList, nameof(uploadList));
```

Comment:
This is not needed. The parameter name in the resulting exception defaults to the string used in the expression, which happens to be exactly the same as what you are passing for the parameter name, It's basically TMI for no benefit.

Reply:
Not sure I agree here. I added the name mostly, because I've seen this pattern in most places in the code base.  If anything, this will result in a consistent implementation.